src/helpers/figures/charts/bar_chart.ts

```diff
@@ -230,9 +230,8 @@ function getBarConfiguration(
       ticks: {
         color: fontColor,
         callback: (value) => {
-          return localeFormat.format
-            ? formatValue(value, localeFormat)
-            : value?.toLocaleString() || value;
+          const { locale, format } = localeFormat;
+          return formatValue(value, { locale, format: !format && value > 1000 ? "#,##" : format });
```

Comment:
this will unfortunately not pass (starting typescript 5.1.6 at least)

Reply:
I'm surprised it worked before. Should be good now